pygccxml/declarations/comment.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2014-2017 Insight Software Consortium.
+# Copyright 2004-2009 Roman Yakovenko.
+# Distributed under the Boost Software License, Version 1.0.
+# See http://www.boost.org/LICENSE_1_0.txt
+
+"""
+Describe a C++ comment declaration.
+
+"""
+
+from . import location as pygccxml_location
+
+
+class comment_t():
```

Comment:
Can you change this to: `comment_t(object)` ?

This is the style I have used everywhere. I think we the newest Python version we can just write `class comment_t`, but to be consistent with the rest of the code base I would like to use `(object)` everywhere.

Reply:
Done.